kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -263,7 +263,7 @@
     position: fixed;
     right: 8px;
     z-index: 8;
-
+    outline: none;
```

Comment:
Why was this added? I see it in a number of places

Reply:
Hey, actually I was working on two different issues simultaneously. These changes are of a different issue.
I believe this issue requires more research. Can I put this issue on hold and take some time understanding it?

Reply:
When you initially suggested this solution to the issue, @jonboiser pointed out that this absolutely was not the approach to take: https://github.com/learningequality/kolibri/issues/7974#issuecomment-815955969

Subsequently you have opened three separate PRs with these changes. Please pay attention to the feedback from repository maintainers so that we do not spend our time reviewing code that you have already been told is not an appropriate fix for the issue at hand.

Reply:
Actually I was having some difficulties. But I understand sorry for this, these PRs were created by mistake. I will close this PR. And look for some other issues and will ask the mentors about this outline one. I am sorry for this.